x-pack/plugins/observability/common/threshold_rule/metric_value_formatter.ts

```diff
@@ -10,7 +10,7 @@ import { createFormatter } from './formatters';
 
 export const metricValueFormatter = (value: number | null, metric: string = '') => {
   const noDataValue = i18n.translate(
-    'xpack.observability.threshold.rule.alerting.noDataFormattedValue',
+    'xpack.observability.customThreshold.rule.alerting.noDataFormattedValue',
```

Comment:
@fkanout I see this file is in a `threshold_rule` folder. Should the folder maybe be renamed to custom threshold or is it more like a common thing?

Reply:
@mgiota, good catch 👍🏻 . Updated